jquery.dataTables/jquery.dataTables.d.ts

```diff
@@ -629,6 +634,13 @@ declare namespace DataTables {
         data(): DataTable;
 
         /**
+        * Iterate over each selected cell, with the function context set to be the cell in question. Since: DataTables 1.10.6
+        *
+        * @param fn Function to execute for every cell selected.
+        */
+        every(fn: (cellRowIdx?: number, cellColIdx?: number, tableLoop?: number, cellLoop?: number) => void): DataTable;
```

Comment:
In JavaScript a function does not have to declare parameters it does not use. These parameters may still be passed.
When you declare a function type `function f(x?: number): void`, you indicate that the *caller* doesn't have to provide x.

Optional parameters are not needed for callbacks, since a function type `(x: number) => number` may still be implemented by a function `() => 0`.